src/TableOfContents.jl

```diff
@@ -198,6 +198,14 @@ const toc_css = """
 .plutoui-toc.aside.hide {
 	transform: translateX(calc(100% - 28px));
 }
+.plutoui-toc.aside.hide section {
+    	display: none;
+}
+.plutoui-toc.aside.hide header {
+    	margin-bottom: 0em;
```

Comment:
This would actually need to be `-0.1em` to be simmetric with the top margin, but I found out that it looks more simmetric like this on my laptopt